packages/ui/components/editor/Editor.test.tsx

```diff
@@ -1,9 +1,55 @@
+import { CodeHighlightNode, CodeNode } from "@lexical/code";
+import { AutoLinkNode, LinkNode } from "@lexical/link";
+import { ListItemNode, ListNode } from "@lexical/list";
+import { LexicalComposer } from "@lexical/react/LexicalComposer";
+import { ContentEditable } from "@lexical/react/LexicalContentEditable";
+import LexicalErrorBoundary from "@lexical/react/LexicalErrorBoundary";
+import { RichTextPlugin } from "@lexical/react/LexicalRichTextPlugin";
+import { HeadingNode, QuoteNode } from "@lexical/rich-text";
+import { TableCellNode, TableNode, TableRowNode } from "@lexical/table";
 import { render, screen } from "@testing-library/react";
+import { fireEvent, waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { describe, it, expect, vi } from "vitest";
 
 import type { TextEditorProps } from "./Editor";
 import { Editor } from "./Editor";
+import ExampleTheme from "./ExampleTheme";
+import { VariableNode } from "./nodes/VariableNode";
+import CustomEnterKeyPlugin from "./plugins/customEnterKeyPlugin";
+
+const editorConfig = {
+  theme: ExampleTheme,
+  onError(error: Error) {
+    throw error;
+  },
+  namespace: "",
+  nodes: [
+    HeadingNode,
+    ListNode,
+    ListItemNode,
+    QuoteNode,
+    CodeNode,
+    CodeHighlightNode,
+    TableNode,
+    TableCellNode,
+    TableRowNode,
+    AutoLinkNode,
+    LinkNode,
+    VariableNode,
+  ],
+};
+
+const TestEditor = () => (
+  <LexicalComposer initialConfig={editorConfig}>
+    <RichTextPlugin
+      contentEditable={<ContentEditable />}
+      placeholder={<div>Enter some text...</div>}
+      ErrorBoundary={LexicalErrorBoundary}
+    />
+    <CustomEnterKeyPlugin />
```

Comment:
Hey, please move the test where CustomEnterKeyPlugin is present and replace the filename editor with CustomEnterKeyPlugin.test.tsx. We have a test file on the way to add for the editor